src/gcp_scanner/scanner.py

```diff
@@ -164,24 +169,31 @@ def get_resources(project: models.ProjectInfo):
     logging.error('Try removing the %s file and restart the scanner.',
                   output_file_name)
 
-  for crawler_name, client_name in CRAWL_CLIENT_MAP.items():
-    if is_set(project.scan_config, crawler_name):
-      crawler_config = {}
-      if project.scan_config is not None:
-        crawler_config = project.scan_config.get(crawler_name)
-      # add gcs output path to the config.
-      # this path is used by the storage bucket crawler as of now.
-      crawler_config['gcs_output_path'] = gcs_output_path
-      # crawl the data
-      crawler = CrawlerFactory.create_crawler(crawler_name)
-      client = ClientFactory.get_client(client_name).get_service(
-        project.credentials,
-      )
-      project_result[crawler_name] = crawler.crawl(
-        project_id,
-        client,
-        crawler_config,
-      )
+  results_crawl_pool = dict()
+  with concurrent.futures.ThreadPoolExecutor(max_workers=int(project.worker_count) + 4) as executor:
```

Comment:
Thanks for implementing that, why +4? Let's make it transparent for the user and give them full control of how many threads/processes they want to execute.

Reply:
![image](https://github.com/google/gcp_scanner/assets/77539004/c17f0f32-7618-45df-b5d5-2c1fb0c6a0b9)

It was because of this :point_up: . This was mentioned in the docs.

https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor

I'll change it up to just `worker_count`.